ibm_catalog.json

```diff
@@ -93,7 +93,366 @@
+                },
+                {
+                  "displayname": "Option 5",
+                  "value": "onerate_active"
+                }
+              ]
+            },
+            {
+              "key": "cos_bucket_archive_enabled"
+            },
+            {
+              "key": "cos_bucket_archive_days"
+            },
+            {
+              "key": "cos_bucket_archive_type",
+              "options": [
+                {
+                  "displayname": "Option 1",
+                  "value": "Glacier"
+                },
+                {
+                  "displayname": "Option 2",
+                  "value": "Accelerated"
+                }
+              ]
+            },
+            {
+              "key": "cos_bucket_retention_default"
+            },
+            {
+              "key": "cos_bucket_retention_enabled"
+            },
+            {
+              "key": "cos_bucket_retention_maximum"
+            },
+            {
+              "key": "cos_bucket_retention_minimum"
+            },
+            {
+              "key": "cos_bucket_retention_permanent"
+            },
+            {
+              "key": "skip_atracker_cos_iam_auth_policy"
+            },
+            {
+              "key": "cos_bucket_management_endpoint_type",
+              "options": [
+                {
+                  "displayname": "Option 1",
+                  "value": "public"
+                },
+                {
+                  "displayname": "Option 2",
+                  "value": "private"
+                },
+                {
+                  "displayname": "Option 3",
+                  "value": "direct"
+                }
+              ]
+            },
+            {
+              "key": "cos_target_name"
+            },
+            {
+              "key": "activity_tracker_route_name"
+            },
+            {
+              "key": "activity_tracker_locations"
+            },
+            {
+              "key": "provision_trusted_profile_projects"
+            },
+            {
+              "key": "trusted_profile_name"
+            },
+            {
+              "key": "trusted_profile_description"
+            },
+            {
+              "key": "trusted_profile_roles"
+            },
+            {
+              "key": "audit_resource_group_name"
+            },
+            {
+              "key": "devops_resource_group_name"
+            },
+            {
+              "key": "edge_resource_group_name"
+            },
+            {
+              "key": "management_resource_group_name"
+            },
+            {
+              "key": "observability_resource_group_name"
+            },
+            {
+              "key": "security_resource_group_name"
+            },
+            {
+              "key": "workload_resource_group_name"
+            },
+            {
+              "key": "existing_audit_resource_group_name",
+              "custom_config": {
```

Comment:
Your not going to be able to use the `resource_group` picker until catalog allow the default value of `null` to be auto selected. If you go with this now, the user is going to have to explicitly select null for all of those resource group picker vars you have, which is a terrible user experience. For now remove all custom config for all the resource group vars